components/pages/MyGarden.js

```diff
@@ -17,29 +17,36 @@ export default class MyGarden extends Component {
 
   getAllGardenPlants = () => {
     api.getAllGardenPlants()
-    .then(response => {
-      this.setState({gardenPlants: response.data})
-    })
-    .catch(err => console.log(err))
+      .then(response => {
+        this.setState({gardenPlants: response.data})
+      })
+      .catch(err => console.log(err))
+  }
+
+  renderGardenPlants = () => {
+    return (
+      this.state.gardenPlants.map( plant => (
+        <GardenPlant
+          navigation={ this.props.navigation }
+          info={plant}
+          key={plant.id}/>
+      ))
+    )
```

Comment:
We were forced to create a conditional for this. It was erroring out when we had no plants in the garden

Reply:
I was knocking my head on this too, awesome figuring it out!!